index.html

```diff
@@ -186,186 +186,224 @@ <h2 class="title features__title">
+              width="145"
+              height="165"
+              alt="illustration-free"
+            />
+            <p class="choose-plan__description">Premium Plan</p>
+            <ul class="choose-plan__list--item">
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Unlimited Bandwitch</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Encrypted Connection</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">No Traffic Logs</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Works on All Devices</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Connect Anyware</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Get New Features</p>
+              </li>
+            </ul>
+            <h3 class="choose-plan__subtitle">
+              $12 <span class="choose-plan__light">/ mo</span>
+            </h3>
+            <button class="button choose-plan__button" type="button">
+              Select
+            </button>
+          </div>
         </div>
       </section>
 
       <section class="global-net">
-        <h2 class="global-net__title">Huge Global Network of Fast VPN</h2>
-        <p class="global-net__subtitle">
+        <h2 class="global-net__title title">Huge Global Network of Fast VPN</h2>
```

Comment:
This headline is split into 2 lines in the design, you could do the same, but let's try to avoid using `<br>` this time. Please use `max-width` (and no-break-space if necessary) instead.

Reply:
++